framework/freedomotic-core/src/test/java/com/freedomotic/rules/StatementTest.java

```diff
@@ -0,0 +1,108 @@
+package com.freedomotic.rules;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class StatementTest {
+
+  @Test
+  public void createLogicalInputValid() {
+    final Statement statement = new Statement();
+    final Statement retval = statement.create(Statement.EQUALS, "retval1", Statement.AND, "100");
+    Assert.assertNotNull(retval);
+    Assert.assertEquals(Statement.EQUALS, retval.getLogical());
+    Assert.assertEquals("retval1", retval.getAttribute());
+    Assert.assertEquals(Statement.AND, retval.getOperand());
+    Assert.assertEquals("100", retval.getValue());
+  }
+
+  @Test
+  public void createLogicalInputInvalid() {
+    final Statement statement = new Statement();
+    final Statement retval = statement.create(Statement.NOT, "retval2", Statement.OR, "100");
```

Comment:
remove the otherwise unused `statement` declaration, inline into line 22 as `retval = new Statement().create(...`